### PR TITLE
Ch10063/trustedform integration staging certificates

### PR DESCRIPTION
**Pull request by jarrmill**
Full ticket info [here](https://app.clubhouse.io/active-prospect/story/10063/trustedform-integration-staging-certificates-should-not-be-skipped-on-lc-staging)

### CODE REVIEW

**Review by jgrayson676 — APPROVED** [review]
LGTM

File: lib/claim.js [hunk 1]
```diff
@@ -51,14 +51,21 @@ const requestVariables = () => [
 ];
 
 const validate = (vars) => {
+  const isNotProduction = (process.env.NODE_ENV === 'staging' || process.env.NODE_ENV === 'development');
```
Comment (kellym) [inline lib/claim.js:54]: I think this is probably better represented by a comparison with `production` 
```suggestion
  const isNotProduction = process.env.NODE_ENV !== 'production';
```

File: test/claim_spec.js [hunk 2]
```diff
@@ -36,6 +40,24 @@ describe('Cert URL validate', () => {
...
+  it('should not accept staging certs on production', () => {
+    const error = integration.validate({lead: { trustedform_cert_url: 'http://cert.staging.trustedform.com/2605ec3a321e1b3a41addf0bba1213505ef57985' }});
+    assert.equal(error, 'TrustedForm cert URL must be valid');
+  });
+
+  it('should accept a production cert on staging', () => {
+    process.env.NODE_ENV = 'staging';
+    const error = integration.validate({lead: { trustedform_cert_url: 'https://cert.trustedform.com/2605ec3a321e1b3a41addf0bba1213505ef57985' }});
+    assert.isUndefined(error);
+  });
```
Comment (cgrayson) [inline test/claim_spec.js:60]: Could add a test for regular old bad certs on staging.

**Review by cgrayson — APPROVED** [review]
Might also remove the Travis badge from README, otherwise LGTM

**Pull request merged**